integration_tests/loop_11.py

```diff
@@ -0,0 +1,41 @@
+from lpython import i32
+
+#checking for loops in the global scope 
+sum: i32 = 0
+i: i32
+for i in [1, 2, 3, 4]:
+    print(i)
+    sum += i
+print("sum = "+str(sum))
```

Comment:
```suggestion
print("sum = ", sum)
```

This is a minor thing. Explicit casting to `str` is not required. Also, let's use a comma instead of the `+` operator. It is typically not used in Python inside `print()`.

Reply:
Ok